driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/recvmsg.bpf.c

```diff
@@ -80,7 +80,7 @@ int BPF_PROG(recvmsg_x,
 
 		/* Parameter 3: data (type: PT_BYTEBUF) */
 		unsigned long msghdr_pointer = extract__syscall_argument(regs, 1);
-		auxmap__store_iovec_data_param(auxmap, msghdr_pointer, bytes_to_read);
+		auxmap__store_msghdr_iovec_data_param(auxmap, msghdr_pointer, bytes_to_read);
```

Comment:
self-review: this change will then be also required in `socketcall` PR #811 (depending which one gets merged earlier).